basil/TL/PrologixVisa.py

```diff
@@ -0,0 +1,53 @@
+#
```

Comment:
As I see it the only difference to the already existing VISA implementation is the query command. So one could simplify this module to:
``` python

class PrologixVisa(Visa):
    def query(self, data):
        self._resource.write("++auto 0")
        self._resource.write(data)
        return self._resource.query("++read eoi")
```

and it would do the same, no?
I am not sure if the _write_ and _read_ commands also need changes. Did you check?